### PR TITLE
add param - MaxAnyAllExpressionDepth

### PR DESCRIPTION
**Pull request by Evgentik**

### CODE REVIEW

File: Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs [hunk 1]
```diff
@@ -32,6 +33,7 @@ public IQueryable<CandidateDTO> Get()
 
         [EnableQuery(
             HandleNullPropagation = HandleNullPropagationOption.False,
+            MaxAnyAllExpressionDepth = 5,
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs:36]: May be 2?

File: Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs [hunk 2]
```diff
@@ -23,6 +23,7 @@ public CandidatesController(ICandidateService candidateService)
 
         [EnableQuery(
             HandleNullPropagation = HandleNullPropagationOption.False,
+            MaxAnyAllExpressionDepth =5,
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs:26]: Spaces and 2

File: Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs [hunk 3]
```diff
@@ -15,12 +15,7 @@ public CandidateProfile()
                 .ForMember(dest => dest.Id, opt => opt.Ignore());
             CreateMap<ProjectResult, ProjectResultDTO>()
                 .ForMember(dest => dest.PrimarySkill, conf => conf.MapFrom(src => new CandidatePrimarySkillDTO { Id = src.PrimarySkill.Id, Name = src.PrimarySkill.Name }))
-                .ForMember(dest => dest.IsAssigned, conf => conf.MapFrom(src => false))
-                .ForMember(dest => dest.TestRating, opt => opt.Ignore())
-                .ForMember(dest => dest.MentorFeedbackRating, opt => opt.Ignore())
-                .ForMember(dest => dest.InterviewRating, opt => opt.Ignore())
-                .ForMember(dest => dest.TechInterviewOneStepRating, opt => opt.Ignore())
-                .ForMember(dest => dest.TechInterviewSecondStepRating, opt => opt.Ignore());
+                .ForMember(dest => dest.IsAssigned, conf => conf.MapFrom(src => false));
```
Comment (elizavetapavliv) [inline Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs:18]: Remove, Alesya removed in here pr

File: Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs (hunk 2 above)
Comment (Evgentik) [reply Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs]: +

File: Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs (hunk 3 above)
Comment (Evgentik) [reply Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs:18]: +

File: Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs (hunk 1 above)
Comment (Evgentik) [reply Recrutify/Recrutify.Host/Controllers/OData/CandidatesController.cs]: +

File: Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs (hunk 3 above)
Comment (elizavetapavliv) [reply Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs:18]: Remove this line

Comment (Evgentik) [reply Recrutify/Recrutify.Host/Configuration/Profiles/CandidateProfile.cs]: +

**Review by NinaLa13 — APPROVED** [review]

**Pull request merged**